src/react/components/pages/editorPage/editorTagsInput.test.tsx

```diff
@@ -117,92 +133,34 @@ describe("Tags Input Component", () => {
         expect(wrapper.find(EditorTagsInput).state().tags).toHaveLength(originalTags.length);
     });
 
-    it("ctrl clicking tag opens editor modal", () => {
```

Comment:
Why are these tests removed?  Seems like a good test to validate that the editor modal open when the take is clicked.

Reply:
those tests are removed because - as tanner confirmed previously - the modal and tags input components are no longer connected to one another. the modal does not call the function on tags input. it calls it on editorFooter.